### ISSUE TITLE
[Help]: jumps after unwrapping

### ISSUE THREAD

**Issue by AmiraZakii** [gmtsar/gmtsar#797]
Hello,
I want to ask why after applying phase unwrapping sometimes some interferograms show these parts of jumps and others are smooth
The first two images represent the unwrapping interferograms with and without jumps. Also, I attached the wrapped phase for the same acquisition
![unwrapped](https://github.com/gmtsar/gmtsar/assets/89259389/ab448054-c64d-4700-97bb-60249c1e2a82)
![unwrapped_jumps](https://github.com/gmtsar/gmtsar/assets/89259389/90176f11-40af-4548-8f0b-9064cdc3fb61)
![phase](https://github.com/gmtsar/gmtsar/assets/89259389/c8835f8c-cedd-4673-bc4a-80d9d421e18a)
![phase_2](https://github.com/gmtsar/gmtsar/assets/89259389/33813eee-bfef-4c0b-816e-77a01b96ef3a)

**Comment by CUMT530849827** [comment]
Hello, I think I have also encountered similar problems. Is it true that dem does not fully cover sar data leading to such problems? Please check it. Of course, this is only my personal opinion

**Comment by mobigroup** [comment]
Yes, the issue looks as topographic phase and it means DEM does not cover the full area. That’s better to always check scenes on DEM and topography in radar coordinates like to the example Google Colab notebook: https://colab.research.google.com/drive/1sljxm2jAMGXynq4EYam6Siz8OLcPLN0h?usp=sharing

**Comment by Xiaohua-Eric-Xu** [comment]
These looks like from incomplete DEM coverage. It could be that when you create the DEM with make_dem.csh, when fetching remote files from U-Hawaii, one piece of tile was missing. I would delete everything in your `~/..gmt/server/earth/earth_relief/earth_relief_??s_g/*.nc` and then rerun make_dem.csh to download these tiles correctly.
